python/ray/serve/controller.py

```diff
@@ -519,6 +538,28 @@ async def __init__(self,
 
         asyncio.get_event_loop().create_task(self.run_control_loop())
 
+    async def wait_for_event(self, uuid: ResultId) -> FutureResult:
+        future = self.inflight_results[uuid]
+        await future.wait()
+        # NOTE(ilr) We have to delete this *after* we return to avoid a
+        # situation where the controller crashes *after* deleting and before
+        # returning to the API (and user).
+        asyncio.get_event_loop().call_soon_threadsafe(
```

Comment:
- we are in this event loop so no need the thread_safe. 
- just call create_task should be fine.

Reply:
`create_task` doesn't seem to get called :/

Reply:
https://travis-ci.com/github/ray-project/ray/jobs/453250888